dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -27,6 +27,8 @@ import (
 
 	"github.com/elastic/beats/v7/libbeat/dashboards"
 	"github.com/elastic/beats/v7/libbeat/kibana"
+	"github.com/elastic/beats/v7/libbeat/version"
+	kbn "github.com/elastic/elastic-agent-libs/kibana"
```

Comment:
Why are we importing `"github.com/elastic/beats/v7/libbeat/kibana"` still?

Reply:
`kibana` package contains dashboard loading/exporting logic. Now that you bring it up, it can be probably moved to `dashboards` package instead.